flink-ml-python/pyflink/ml/core/tests/test_stage.py

```diff
@@ -21,14 +21,16 @@
 from pyflink.table import StreamTableEnvironment
 
 from pyflink.ml.core.api import Stage
+from pyflink.ml.core.linalg import Vectors
 from pyflink.ml.core.param import ParamValidators, Param, BooleanParam, IntParam, \
-    FloatParam, StringParam, IntArrayParam, FloatArrayParam, StringArrayParam
+    FloatParam, StringParam, VectorParam, IntArrayParam, FloatArrayParam, StringArrayParam
 from pyflink.ml.tests.test_utils import PyFlinkMLTestCase
 
 BOOLEAN_PARAM = BooleanParam("boolean_param", "Description", False)
 INT_PARAM = IntParam("int_param", "Description", 1, ParamValidators.lt(100))
 FLOAT_PARAM = FloatParam("float_param", "Description", 3.0, ParamValidators.lt(100))
 STRING_PARAM = StringParam('string_param', "Description", "5")
+VECTOR_PARAM = VectorParam('vector_param', "Description", Vectors.dense(1, 2, 3))
```

Comment:
It might be unable to verify the correctness of `VectorParam` by just creating the `VECTOR_PARAM`. Could you please refer to tests for other parameters, like `IntParam`, and add corresponding test cases about `VectorParam`?